mysql-test/suite/rocksdb_sys_vars/r/all_vars.result

```diff
@@ -9,5 +9,7 @@ There should be *no* long test name listed below:
 select variable_name as `There should be *no* variables listed below:` from t2
 left join t1 on variable_name=test_name where test_name is null ORDER BY variable_name;
 There should be *no* variables listed below:
+ROCKSDB_MRR_BATCH_SIZE
+ROCKSDB_MRR_BATCH_SIZE
```

Comment:
No, this is not the right way to do it. As the above line says, "There should be *no* variables listed below"

Reply:
This is how to fix it: https://gist.github.com/spetrunia/0e2cd3359990e9d14705aaf24f57baac

Reply:
shame on myself. let me fix it in another PR.

Reply:
Use https://github.com/facebook/mysql-5.6/pull/1085 to track this issue.